### DISCUSSION TITLE
Is there a way to get new dimensions if the user pinch zooms or shrinks the model?

### DISCUSSION THREAD

**Discussion by streetcodec** [google/model-viewer#4349] [Q&A] [answered]
I thought I can get new dimensions via .getDimensions() but obviously that won't work, because glb themselves aren't get big or small.
I want to have a button that, when clicked, will restore the image to the original size.

**Comment by elalish** [comment, marked as answer]
Dimensions are mostly useful in AR, as GLBs are in real world units of meters. For plain 3D, perhaps you mean reset the zoom? You can use `mv.zoom(-100);` https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-methods-zoom to zoom all the way back to the stops (initial).

**Reply by streetcodec** [reply]
Okay, thanks!
A follow up question, can we get the data about how much the user zoomed in or out the painting?

**Reply by elalish** [reply]
Sure, try `mv.getCameraOrbit().radius`.
